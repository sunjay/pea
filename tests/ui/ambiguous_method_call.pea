fn main() {
    let value = if true {
        return;
    } else {
        return;
    };

    ((((value)))).foo();
}
