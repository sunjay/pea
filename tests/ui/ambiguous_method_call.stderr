[tests/ui/ambiguous_method_call.pea:8:5-8:17] error: the type of this value must be known in this context

error: aborting due to 1 previous error

